pages/items/new.tsx

```diff
@@ -0,0 +1,47 @@
+import attributesHandler from '@api/attributes'
+import categoriesHandler from '@api/categories'
+import {
+  Button,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+} from '@mui/material'
+import UpsertItemForm from 'components/UpsertItemForm'
+import { useRouter } from 'next/router'
+import React from 'react'
+import { AttributeResponse, CategoryResponse } from 'utils/types'
+let categories: CategoryResponse[]
+let attributes: AttributeResponse[] = [] as AttributeResponse[]
+fetch('http://localhost:3000/api/categories', {
```

Comment:
this code will be called on the frontend, whereas the `apiWrapper` code is meant to run on the server. the `getServerSideProps` function is run on the server, but since this isn't in that function it must use fetch

Reply:
still don't really understand the difference but sure lol